Bio/SearchIO/BlastIO/blast_tab.py

```diff
@@ -569,9 +568,9 @@ def _qresult_index_commented(self):
         # mark of a new query
         query_mark = None
         # mark of the query's ID
-        qid_mark = _as_bytes("# Query: ")
+        qid_mark = as_bytes("# Query: ")
```

Comment:
`b"# Query: "` and similar for the other cases.

Reply:
Yes, all ``as_bytes("example")`` to ``b"example"`` please.

From memory we probably had to use ``as_bytes("example")`` back when we supported Python 2.5. We could have done this cleanup a while ago since Python 2.6+ onwards supports bytes literals.

Reply:
Agreed, changed it, including one instance of regexp.